plugins/destination/gcs/client/client.go

```diff
@@ -22,6 +22,7 @@ type Client struct {
 	streamingbatchwriter.UnimplementedDeleteStale
 	streamingbatchwriter.UnimplementedDeleteRecords
 
+	syncID string
```

Comment:
SyncID needs to be set in `client.New()`

You can see how it is done in the S3 destination: https://github.com/cloudquery/cloudquery/blob/9b07ab328eb8b3ef7fab61c9d8458244cc33af93/plugins/destination/s3/client/client.go#L44

Reply:
Thanks!

Pushed the changes at https://github.com/cloudquery/cloudquery/compare/c2b3f14b168cd5d8b9f8c5a8474ee94a7c50ea5b..693d590150666a0282a090934885d80fd34d5df9

Test sync on GCS with path as `new/{{TABLE}}/{{UUID}}_{{SYNC_ID}}.{{FORMAT}}`

![Screenshot 2024-04-12 at 2 02 00 AM](https://github.com/cloudquery/cloudquery/assets/13910561/18fc4cf2-959c-488b-aa3b-3d149e6b47fd)